tests/hlsl-intrinsic/texture/texture-intrinsics.slang

```diff
@@ -244,6 +246,28 @@ void computeMain(int3 dispatchThreadID: SV_DispatchThreadID)
     f4 = t2DArray.Gather(samplerState, float3(u, u, 0), int2(0, 0));
     val += f4.x;  val += f4.y;  val += f4.z;  val += f4.w;
 
+    // Object.GatherGreen()
+    f4 = t2D.GatherGreen(samplerState, float2(u, u), int2(1, 2));
+    val += f4.x;  val += f4.y;  val += f4.z;  val += f4.w;
+
+    f4 = t2D.GatherGreen(samplerState, float2(u, u), int2(1, 2), int2(3, 4), int2(5, 6), int2(7, 8));
+    val += f4.x;  val += f4.y;  val += f4.z;  val += f4.w;
+
+    f4 = tCubeArray.GatherGreen(samplerState, float4(1.5, 1.5, 1.5, 1.5));
+    val += f4.x;  val += f4.y;  val += f4.z;  val += f4.w;
+
+    // status variant
+#if !defined(VK)
+    f4 = t2D.GatherGreen(samplerState, float2(u, u), int2(1, 2), status);
+    val += f4.x;  val += f4.y;  val += f4.z;  val += f4.w;
+
+    f4 = t2D.GatherGreen(samplerState, float2(u, u), int2(1, 2), int2(3, 4), int2(5, 6), int2(7, 8), status);
+    val += f4.x;  val += f4.y;  val += f4.z;  val += f4.w;
+
+    f4 = tCubeArray.GatherGreen(samplerState, float4(1.5, 1.5, 1.5, 1.5), status);
+    val += f4.x;  val += f4.y;  val += f4.z;  val += f4.w;
+#endif
```

Comment:
@ArielG-NV , I think there is a bug on the capability check.
When `#if !defined(VK)` is not there, VK test ran just fine, which should have triggered a capability error.
Can you take a look at this?